doc/man3/CRYPTO_get_ex_new_index.pod

```diff
@@ -17,12 +17,12 @@ CRYPTO_get_ex_data, CRYPTO_free_ex_data, CRYPTO_new_ex_data
                 CRYPTO_EX_dup *dup_func,
                 CRYPTO_EX_free *free_func);
 
- typedef int CRYPTO_EX_new(void *parent, void *ptr, CRYPTO_EX_DATA *ad,
-                           int idx, long argl, void *argp);
- typedef void CRYPTO_EX_free(void *parent, void *ptr, CRYPTO_EX_DATA *ad,
-                             int idx, long argl, void *argp);
- typedef int CRYPTO_EX_dup(CRYPTO_EX_DATA *to, const CRYPTO_EX_DATA *from,
-                           void *from_d, int idx, long argl, void *argp);
+ typedef int (*CRYPTO_EX_new)(void *parent, void *ptr, CRYPTO_EX_DATA *ad,
+                              int idx, long argl, void *argp);
+ typedef void (*CRYPTO_EX_free)(void *parent, void *ptr, CRYPTO_EX_DATA *ad,
+                                int idx, long argl, void *argp);
+ typedef int (*CRYPTO_EX_dup)(CRYPTO_EX_DATA *to, const CRYPTO_EX_DATA *from,
+                              void *from_d, int idx, long argl, void *argp);
```

Comment:
WTF?  that seems wrong.  all our other callback types have the parentheses!  is this just a bug?  or am I a C neophyte?